src/cli/cli.cpp

```diff
@@ -8780,6 +8861,7 @@ otError Interpreter::ProcessCommand(Arg aArgs[])
         CmdEntry("csl"),
 #endif
         CmdEntry("dataset"),
+        CmdEntry("debug"),
```

Comment:
Should we add the command usage to `README.md`?

Reply:
I intended to keep this a hidden command, for experts and folks that want to debug issues. 

I am okay adding short description, but I don't think it would be useful to describe what this command does in detail (and/or provide output samples as it may get changed).

Reply:
Added documentation in both `cli/README.md` and `"debug"` command definition in new pushed commit.